README.md

```diff
@@ -1976,7 +1976,7 @@ Equivalent to [Plot.vector](#plotvectordata-options) except that the **shape** d
 
 ## Decorations
 
-Decorations are static marks that do not represent data. Currently this includes only [Plot.frame](#frame), although internally Plot’s axes are implemented as decorations and may in the future be exposed here for more flexible configuration.
+Decorations are static marks that do not represent data. Currently this includes [Plot.frame](#frame), as well as the [axes](#axis) and [grid](#grid) marks described above.
```

Comment:
It feels better to keep them documented here even if they don't exactly fit the description. Alternatively, remove the whole paragraph?

Reply:
I guess this is still arguably accurate because it says the marks “do not represent data”—I first read it as “do not _have_ data”. The word “static” is a little misleading though, because that sounds like it’s one instance per mark (as with frame), which isn’t true with axes and grids.

Reply:
```suggestion
Decorations are marks that do not directly represent data. Currently this includes [Plot.frame](#frame), as well as the [axes](#axis) and [grid](#grid) marks described above.
```